device-types/Datacom/DM4770-32CX.yaml

```diff
@@ -0,0 +1,132 @@
+---
+manufacturer: Datacom
+model: DM4770 32CX
+slug: datacom-dm4770-32cx
+part_number: 800.5255.xx
+u_height: 1.0
+is_full_depth: true
+airflow: side-to-rear
+comments: Switch with L2, L3, and MPLS capabilities, with 32 40GE/100GE interfaces on QSFP28 connectors and 2 GE/10GE interfaces on SFP+ connectors. 1U
+  metal enclosure for 19 rack with 4 slots for fan modules and two slots for redundant AC or DC power supplies. Outband management interfaces, RS232 console,
+  and USB ports. Power supplies, FAN modules, MPLS license, and optical modules must be purchased separately.
+weight: 6.3
+weight_unit: kg
+console-ports:
+  - name: Console0
+    type: rj-45
+  - name: Console1
+    type: usb-mini-b
+console-server-ports:
+  - name: Storage
+    type: usb-a
```

Comment:
Is this actually used as a console server port? Or is it just for the switch to access a USB storage device? If the latter, please remove this definition as it would be out of scope.